tests/test_http.py

```diff
@@ -26,6 +27,11 @@ def s():
 
 def has_connection():
     # In case we run tests in an environment without internet connection.
+
+    if sys.version_info < (2, 7):
+        # py 2.6 doesn't have SNI support, required for httpbin, as it has SSLv3 certificates
+        return False
```

Comment:
What I'm saying is: SNI may cause diagnostic output, but does it actually cause tests to fail?

So far I'm not convinced that it does. We weren't seeing "hostname doesn't match" errors -- we were seeing an "internal error".

Could we at least try leaving the tests on for Python 2.6, but with the requests.__version__ check in place?